witchcraft-logging-idea/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin url="https://github.com/palantir/witchcraft-api">
     <id>com.palantir.witchcraft.api.logging.idea</id>
     <name>Witchcraft Logging</name>
-    <version>0.0.0</version>
+    <description>Automatically renders witchcraft-logging-api structured log events from the idea console into color-coded human-readable text.</description>
```

Comment:
emphasis on 'automatic'  rather than 'render' so it's clear there's no button you have to click to make it work.